packages/rollup-plugin/src/index.js

```diff
@@ -25,6 +27,11 @@ const IS_DEV_ENV =
   process.env.NODE_ENV === 'development' ||
   process.env.BABEL_ENV === 'development';
 
+type LightningcssTransformOptions = $ReadOnly<{
```

Comment:
@nmn We currently need to define this explicitly because Flow complains if I:
```
import type { TransformOptions } from 'lightningcss';
```

and use it like:
```
export type PluginOptions = $ReadOnly<{
  ...Partial<Options>,
  fileName?: string,
  babelConfig?: $ReadOnly<{
    plugins?: $ReadOnlyArray<PluginItem>,
    presets?: $ReadOnlyArray<PluginItem>,
  }>,
  useCSSLayers?: boolean,
  lightningcssOptions?: TransformOptions<{}>,      <--------Using the type
  ...
}>;
```


Do you have any suggestions to do this more elegantly?